### PR TITLE
Add Travis.yml file

### PR DESCRIPTION
**Pull request by jyeung-r7**

### CODE REVIEW

**Review by mgarewal-r7 — APPROVED** [review]
Looks good :)

File: src/utils.py [hunk 1]
```diff
@@ -86,8 +88,8 @@ class ServerHTTPSConnection(http.client.HTTPSConnection):
     def __init__(self, config, server, port, cert_file):
         self.no_ssl = config.suppress_ssl or not FEAT_SSL
         if self.no_ssl:
-            if config.use_proxy == True:
-                http.client.HTTPSConnection.__init__(self, config.proxy_url, config.proxy_port, context=context)
```
Comment (jsouthern-r7) [inline src/utils.py:90]: By removing the assignment of the context, in this constructor, does it compromise the security of the HTTPS connection in any way?

See:

- https://docs.python.org/3.4/library/http.client.html#http.client.HTTPSConnection
- https://docs.python.org/3.4/library/ssl.html#ssl-security

Comment (jyeung-r7) [reply src/utils.py:90]: @jsouthern-r7 I wondered the same thing. 
But in this case the if block is no_ssl and context wasn't defined in this case. 
Which is why I removed context.

Comment (jsouthern-r7) [reply src/utils.py:90]: I see now. Looks good to me.

**Review by jsouthern-r7 — APPROVED** [review]
ship it

**Pull request merged**